src/components/pagination.tsx

```diff
@@ -13,6 +13,7 @@ interface Props {
   query?: Record<string, unknown>
   pageLinkBuilder?: (page: number, params: Record<string, unknown>) => string
   renderHead?: (links) => void
+  onButtonClick?: (label) => void
```

Comment:
from an interface-perspective, I would not care what underlying elements are used. shall we simplify to 
```suggestion
  onClick?: (label) => void
```

or is it worth being so specific? maybe there's something I don't see...

Reply:
Yea, `onClick` fits better indeed